src/herder/TransactionQueue.cpp

```diff
@@ -771,12 +773,17 @@ TransactionQueue::removeApplied(Transactions const& appliedTxs)
                     if (transaction->mTx->getFullHash() ==
                         appliedTx->getFullHash())
                     {
-                        auto elapsed = now - transaction->mInsertionTime;
-                        mQueueMetrics->mTransactionsDelay.Update(elapsed);
+                        auto elapsed = std::chrono::duration_cast<
```

Comment:
tx queue reasons about txs on like, seconds scale. I think milliseconds is more appropriate here